src/__tests__/helpers/user-agents.json

```diff
@@ -33,10 +33,13 @@
   "iPodWebview": "Mozilla/5.0 (iPod; CPU OS 5_1 like Mac OS X) AppleWebKit/534.46 (KHTML, like Gecko) Mobile/3A101a",
   "iPhone_3_2Safari": "Mozilla/5.0 (iPhone; U; CPU iPhone OS 3_2 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Version/4.0.4 Mobile/7B314 Safari/531.21.10",
```

Comment:
`iPhone_3_2Safari` currently breaks the regex here which is interesting since this version of the OS, 3.2, was from the first generation iPads released in 2010. 
Consider dropping support, if that is allowed given usage, or modifying approach.

Reply:
We should probably do some Splunk-foo to see the usage of `iPhone_3_2Safari` but we do call out in our docs that [we only support Safari 8+](https://braintree.github.io/braintree-web/current/#browser-support-mobile).